### PR TITLE
Sdk improvements

### PR DESCRIPTION
**Pull request by kp-cat**
### Describe the purpose of your pull request

This PR applies the following improvements:
- The evaluation logs are staying on `info` level but they are now combined into a single log entry.
- A `get_all_values()` function was added that evaluates all flags & settings in a dictionary.
- New `flag_overrides` parameter to support the reading of flags & settings from a dictionary.

### Related issues (only if applicable)

https://trello.com/c/FuPidQQt/160-sdk-loglevel-update-python
https://trello.com/c/i4HzNG0Z/105-sdk-getallvalues-python
https://trello.com/c/v1RVkza1/147-sdk-improvements-python

### Requirement checklist (only if applicable)

- [x] I have covered the applied changes with automated tests.
- [x] I have executed the full automated test set against my changes.
- [x] I have validated my changes against all supported platform versions.
- [x] I have read and accepted the [contribution agreement](https://github.com/configcat/legal/blob/main/contribution-agreement.md).

### CODE REVIEW

File: configcatclient/configfetcher.py [hunk 1]
```diff
@@ -49,10 +49,12 @@ def is_not_modified(self):
 
 class ConfigFetcher(object):
     def __init__(self, sdk_key, mode, base_url=None, proxies=None, proxy_auth=None,
-                 data_governance=DataGovernance.Global):
+                 connect_timeout=10, read_timeout=30, data_governance=DataGovernance.Global):
```
Comment (z4kn4fein) [inline configcatclient/configfetcher.py:52]: I think we can trait `connect_timeout` and `read_timeout` parameters required here without default values as those are already defined at the initialization of the client. We may should keep them at a single place to maintain.

File: configcatclient/localdictionarydatasource.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
+from .overridedatasource import OverrideDataSource
+from .constants import *
+
+
+class LocalDictionaryDataSource(OverrideDataSource):
```
Comment (z4kn4fein) [inline configcatclient/localdictionarydatasource.py:5]: Shouldn't we provide a local file data source too for python? WDYT?

**Review by z4kn4fein — CHANGES_REQUESTED** [review]

File: configcatclient/localdictionarydatasource.py (hunk 2 above)
Comment (kp-cat) [reply configcatclient/localdictionarydatasource.py:5]: I'll add it to this PR

File: configcatclient/configfetcher.py (hunk 1 above)
Comment (kp-cat) [reply configcatclient/configfetcher.py:52]: Discussed. We will keep the default values here.

**Review by laliconfigcat — APPROVED** [review]
There are some missing unit tests based on Sonar. Maybe worth checking them, but as I see those are not that important.

**Review by z4kn4fein — APPROVED** [review]

**Pull request merged**